content/en/tracing/manual_instrumentation/dotnet.md

```diff
@@ -51,11 +84,101 @@ catch(Exception e)
 }
 ```
 
+This will set 3 tags on the span, `"error.msg":exception.Message`,   `"error.stack":exception.ToString()`, and `"error.type":exception.GetType().ToString()`.
```

Comment:
Sorry, I missed these in the first pass.
```suggestion
This will set 3 tags on the span: `"error.msg":exception.Message`,  `"error.stack":exception.ToString()`, and `"error.type":exception.GetType().ToString()`.
```